agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/exporter/Exporter.java

```diff
@@ -388,15 +391,23 @@ private void trackTraceAsException(SpanData span, String errorStack) {
   }
 
   private static void setOperationTags(TelemetryItem telemetry, SpanData span) {
-    setOperationTags(telemetry, span.getTraceId(), span.getParentSpanContext().getSpanId());
+    setOperationTags(
```

Comment:
can we simply pass span object? looks like it will require signature change again when something new is required from the span.

Reply:
the more general signature is needed from a couple of places in this class